### PR TITLE
README: use environment variable for image path

### PR DESCRIPTION
**Pull request by blms**
Minor update to the README to reuse the `DPDASH_IMG` environment variable instead of retyping the path of the image file.

### CODE REVIEW

File: README-PNL.md [hunk 1]
```diff
@@ -82,13 +82,13 @@ You can read the [Cookies](#cookies) section for details.
 Launch a DPdash instance as follows:
     
     export DPDASH_IMG=/path/to/dpdash.sif
```
Comment (tashrifbillah) [inline README-PNL.md]: Delete this line

File: README-PNL.md [hunk 2]
```diff
@@ -82,13 +82,13 @@ You can read the [Cookies](#cookies) section for details.
...
-    singularity run -B ${state}:/data -B ${data}:/project_data /path/to/dpdash.sif /sw/apps/dpdash/singularity/run.sh
+    singularity run -B ${state}:/data -B ${data}:/project_data ${DPDASH_IMG} /sw/apps/dpdash/singularity/run.sh
     
 Alternatively, you can shell into the container and then execute further commands. In fact, this would be 
 the recommended method for working with the image.
     
     export DPDASH_IMG=/path/to/dpdash.sif
```
Comment (tashrifbillah) [inline README-PNL.md]: Delete this line

**Review by tashrifbillah — CHANGES_REQUESTED** [review]
Hi Ben, upon reviewing the code base, I remember that only `init.sh` script makes use of `DPDASH_IMG` variable:

```
    export DPDASH_IMG=/path/to/dpdash.sif
    cd dpdash/singularity
    ./init.sh ${data} ${state}
```

Other than the above, we should remove all definitions of `DPDASH_IMG`. We don't require a user to define `DPDASH_IMG` to `run.sh`. So please delete those redundancies only and we should be all set.

**Comment by blms** [comment]
@tashrifbillah I wonder if we should just move the definition of DPDASH_IMG into the "define" section, like so?

> ### Define
> 
> Define two variables `state` and `data` that can be used respectively for DPdash application data and project data, as well as a third variable `DPDASH_IMG` that stores the path of the dpdash.sif image file:
> 
>     export state=/where/you/want/to/save/app/data
>     export data=/where/you/have/PHOENIX/format/project/data
>     export DPDASH_IMG=/path/to/dpdash.sif
> 
> 
> `${state}` is an empty directory that will be populated with files once you do `./init.sh` but you need to have 
> [PHOENIX style directory trees](http://docs.neuroinfo.org/dpdash/en/latest/quick_start.html#create-data-persistence-directories) inside `${data}`.
>

File: README-PNL.md [hunk 3]
```diff
@@ -54,19 +54,19 @@ into the database, instantiate DPdash, and run it.
...
+Define two variables `state` and `data` that can be used respectively for DPdash application
+data and project data, as well as a third variable `DPDASH_IMG` that stores the path of the dpdash.sif image file::
 
     export state=/where/you/want/to/save/app/data
     export data=/where/you/have/PHOENIX/format/project/data
+    export DPDASH_IMG=/path/to/dpdash.sif
 
 
 `${state}` is an empty directory that will be populated with files once you do `./init.sh` but you need to have 
 [PHOENIX style directory trees](http://docs.neuroinfo.org/dpdash/en/latest/quick_start.html#create-data-persistence-directories) inside `${data}`.
```
Comment (tashrifbillah) [inline README-PNL.md:65]: Bring the new direction to a separate paragraph here.

**Comment by tashrifbillah** [comment]
Add `wget` in the Download link

**Comment by tashrifbillah** [comment]
https://github.com/PREDICT-DPACC/dpdash/commit/7368797a9b7a63bd08e572ad7bc4104b1c7002fc

File: README-PNL.md (hunk 3 above)
Comment (blms) [reply README-PNL.md:65]: Something like this? 3cb7ea2aea77c971cf2cb6f49460997e0fee0989

**Comment by tashrifbillah** [comment]
Thank you for your first contribution to PNL software development. After moving `DPDASH_IMG` definition to `Define` section, things are more consistent.

**Pull request merged**